app/src/androidTest/java/net/kuama/documentscanner/domain/FindPaperSheetContoursInstrumentedTest.kt

```diff
@@ -31,20 +31,20 @@ class FindPaperSheetContoursInstrumentedTest {
                     if (resultingCorners != null) {
```

Comment:
```suggestion
    fun findPaperSheetUseCaseValidInputReturnCorrectResult() {
        TestUtils.loadOpenCvLibrary()
        val inputStream: InputStream? = this.javaClass.classLoader?.getResourceAsStream("document-picture.jpg")
        val bitmap = BitmapFactory.decodeStream(inputStream)
        runBlocking {
            findPaperSheetUseCase(
                FindPaperSheetContours.Params(bitmap)
            ) { either ->
                either.mapRight { resultingCorners ->
                    if (resultingCorners != null) {
```

Reply:
there is a way to write the name of test functions as free text between two ' '

Reply:
also use snake case

Reply:
https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods